packages/envied/README.md

```diff
@@ -120,6 +122,9 @@ abstract class Env {
     static const key1 = _Env.key1;
     @EnviedField()
     static const KEY2 = _Env.KEY2;
+    @Default('test_')
```

Comment:
I think, this still needs to be changed. Apart from that, I have not tested the new code yet, but it looks fine now :)

Reply:
I will push the new changes as soon as I tested. I am testing on a different branch.

Reply:
@ThexXTURBOXx @petercinibulk @techouse @daiki1003 if you would like to check it out  you can check over [here](https://github.com/kaankoken/envied/tree/feature/test-default-value).